app/javascript/js/controllers.js

```diff
@@ -53,59 +53,78 @@ import ToggleController from './controllers/toggle_controller'
 import TrixBodyController from './controllers/trix_body_controller'
 import TrixFieldController from './controllers/fields/trix_field_controller'
 
-application.register('action', ActionController)
-application.register('actions-overflow', ActionsOverflowController)
-application.register('actions-picker', ActionsPickerController)
-application.register('attachments', AttachmentsController)
-application.register('boolean-filter', BooleanFilterController)
-application.register('card-filters', CardFiltersController)
-application.register('clear-input', ClearInputController)
-application.register('copy-to-clipboard', CopyToClipboardController)
-application.register('dashboard-card', DashboardCardController)
-application.register('date-time-filter', DateTimeFilterController)
-application.register('filter', FilterController)
-application.register('form', FormController)
-application.register('hidden-input', HiddenInputController)
-application.register('input-autofocus', InputAutofocusController)
-application.register('item-select-all', ItemSelectAllController)
-application.register('item-selector', ItemSelectorController)
-application.register('loading-button', LoadingButtonController)
-application.register('media-library-attach', MediaLibraryAttachController)
-application.register('media-library', MediaLibraryController)
-application.register('menu', MenuController)
-application.register('modal', ModalController)
-application.register('multiple-select-filter', MultipleSelectFilterController)
-application.register('avo-nested-form', NestedFormController)
-application.register('panel-refresh', PanelRefreshController)
-application.register('per-page', PerPageController)
-application.register('preview', PreviewController)
-application.register('record-selector', RecordSelectorController)
-application.register('resource-edit', ResourceEditController)
-application.register('resource-index', ResourceIndexController)
-application.register('resource-show', ResourceShowController)
-application.register('search', SearchController)
-application.register('select-filter', SelectFilterController)
-application.register('select', SelectController)
-application.register('self-destroy', SelfDestroyController)
-application.register('sidebar', SidebarController)
-application.register('sign-out', SignOutController)
-application.register('table-row', TableRowController)
-application.register('tabs', TabsController)
-application.register('text-filter', TextFilterController)
-application.register('tippy', TippyController)
-application.register('toggle', ToggleController)
-application.register('trix-body', TrixBodyController)
+// Special cases for controller registration names
+const SPECIAL_CASES = {
+  nested_form: 'avo-nested-form'
```

Comment:
Is there a reason to not make this like the others and eliminate the special cases?

Reply:
or, change the name that we can change

Reply:
I'm sorry could you rephrase? I didn't get your question